examples/criteo-example.ipynb

```diff
@@ -255,27 +327,27 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 14,
    "metadata": {},
    "outputs": [],
    "source": [
-    "train_data = nvt.Dataset(train_paths, engine=\"parquet\", part_mem_fraction=0.02)\n",
-    "valid_data = nvt.Dataset(valid_paths, engine=\"parquet\", part_mem_fraction=0.02)"
+    "train_data = nvt.Dataset(train_paths, engine=\"parquet\", part_mem_fraction=0.04)\n",
+    "valid_data = nvt.Dataset(valid_paths, engine=\"parquet\", part_mem_fraction=0.04)"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 15,
    "metadata": {},
    "outputs": [],
    "source": [
-    "train_data_itrs = TorchAsyncItr(train_data, batch_size=BATCH_SIZE, cats=CATEGORICAL_COLUMNS, conts=CONTINUOUS_COLUMNS, labels=LABEL_COLUMNS, num_parts=NUM_PARTS)\n",
-    "valid_data_itrs = TorchAsyncItr(valid_data, batch_size=BATCH_SIZE, cats=CATEGORICAL_COLUMNS, conts=CONTINUOUS_COLUMNS, labels=LABEL_COLUMNS, num_parts=NUM_PARTS)"
+    "train_data_itrs = TorchAsyncItr(train_data, batch_size=BATCH_SIZE, cats=CATEGORICAL_COLUMNS, conts=CONTINUOUS_COLUMNS, labels=LABEL_COLUMNS)# , devices=[1,2,3,4])\n",
```

Comment:
I think this might be a bad merge with Julio's recent changes?

Reply:
yeah I had already added a `parts_per_chunk` kwarg, Julio added a similar one just in case my PR didn't make it in in time. Happy to switch syntax or to adjust this example using the new kwarg.

Reply:
yeah, we should have the other version ```num_parts=NUM_PARTS```. Julio added to make the integration tests work automatically (https://github.com/NVIDIA/NVTabular/pull/247).  

There are also some other minor changes with docstrings from the PT classes (https://github.com/NVIDIA/NVTabular/pull/242) and for handling the case where the async thread threw an exception causing a hang (https://github.com/NVIDIA/NVTabular/pull/246).  We should also get these merged back in.